step.yml

```diff
@@ -1,30 +1,35 @@
 title: Script
 summary: |-
-  The power is in your hands. Use it wisely!
+  Run any custom script you want. The power is in your hands. Use it wisely!
 description: |-
-  Specify the script content and (optionally) the script runner,
-  to run any custom script in your [bitrise.io](https://www.bitrise.io/) workflow!
 
-  The **runner** can even be a command, as long as it's available in the `PATH`
-  and can execute the script, it will work.
+  Run any custom script you want as part of your Bitrise build. All you need to do is specify the script runner (the default is bash), add the script, and run a build. You can also read the script from a file.
```

Comment:
Please break the line into multiple shorter ones. For example, you may break it after `specify the script runner`.